tmt/steps/execute/internal.py

```diff
@@ -226,13 +226,21 @@ def _setup_reboot(self, guest):
                     else:
                         # Unrelated error, re-raise
                         raise error
-                guest.execute(f'cp "{reboot_script_path}" "{reboot_file}" && '
-                              f'chmod +x "{reboot_file}"')
+                try:
+                    guest.execute(f'cp "{reboot_script_path}" "{reboot_file}" && '
+                        f'chmod +x "{reboot_file}"')
+                except tmt.utils.RunError as error:
+                    if "Read-only file system" not in error.stderr:
+                        raise error
             yield
         finally:
             self.debug("Remove our reboot script implementations.", level=2)
             for reboot_file in REBOOT_SCRIPT_PATHS:
-                guest.execute(f'rm "{reboot_file}"')
+                try:
+                    guest.execute(f'rm "{reboot_file}"')
+                except tmt.utils.RunError as error:
+                    if error.returncode != 1:
```

Comment:
Why are you checking specifically for exit code `1` here? In the docs I see

```
An exit status of zero indicates success, and a nonzero value indicates failure.
```

Is the status `1` defined for certain types of errors? Or should we just ignore unsuccessful removals?

Reply:
RC 1 should be permission denied OS error but I would guess most other errors would also be indicated by 1.
However I wanted to keep the smallest impact I could, but of course could also be just `except: pass`.

Reply:
Yeah, I've tried it. Seems the exit code `1` is used for non-existent files as well. So... probably does not make sense to check specifically for it. Let's go with `except: pass` unless anyone has objections. @FrNecas thoughts?

Reply:
@psss It makes sense to me what you're saying.

Could we add this change so we unblock the tests on OSTree based images? I guess we could try to find a better solution, but right now this is blocking all our tests :disappointed: 
Thanks!